android/src/main/java/org/mozilla/mozstumbler/client/mapview/MapFragment.java

```diff
@@ -424,9 +423,10 @@ void setHighBandwidthMap(boolean isHighBandwidth) {
         final ClientPrefs.MapTileResolutionOptions tileType = prefs.getMapTileResolutionType();
         final int idxTileType = tileType.ordinal();
         if (idxTileType > 0) {
```

Comment:
not your code I know, but should be
if (tileType != ClientPrefs.MapTileResolutionOptions.Default) {